scripts/offset_example.R

```diff
@@ -0,0 +1,6 @@
+# example
+corunaroads <- readRDS("./data/corunaroads.rds")
```

Comment:
I am not able to install your PR because of the .rds file. Could you either save it as .rda file or maybe just create some lines from sampled coordinates.

> file 'corunaroads.rda' has magic number 'X'
>   Use of save versions prior to 2 is deprecated

Also I see that your example uses `readRDS("./data/corunaroads.rds")` but the PR has only ` data/corunaroads.rda`?

Reply:
I found out that the `rda`  file was a badly renamed `rds` file hence the error. I've managed to fix it in the latest commit.